addon/server.js

```diff
@@ -9,6 +10,27 @@ import _isArray from 'lodash/lang/isArray';
 import _keys from 'lodash/object/keys';
 import _pick from 'lodash/object/pick';
 
+function createPretender(shouldLog) {
+  return new Pretender(function() {
+    this.handledRequest = function(verb, path, request) {
+      if (shouldLog) {
+        console.log('Successful request: ' + verb.toUpperCase() + ' ' + request.url);
```

Comment:
Would it be a better approach to use a template string here, and on L24's error message?